addons/mail/models/ir_http.py

```diff
@@ -13,7 +13,10 @@ def session_info(self):
         """Override to add the current user data (partner or guest) if applicable."""
         result = super().session_info()
         store = StoreData()
-        self.env["res.users"]._init_store_data(store)
+        ResUser = self.env["res.users"]
```

Comment:
Whether we like it or not, actual class name is `ResUsers` with `s` at the end though.